crates/bevy_core_pipeline/src/tonemapping/tonemapping_shared.wgsl

```diff
@@ -1,5 +1,15 @@
 #define_import_path bevy_core_pipeline::tonemapping
 
+// from https://knarkowicz.wordpress.com/2016/01/06/aces-filmic-tone-mapping-curve
+fn aces_filmic(color: vec3<f32>) -> vec3<f32> {
```

Comment:
Remainder that now with https://github.com/bevyengine/bevy/pull/7594, this should be removed.

Reply:
Perhaps we can merge this and then further improve tonemapping in a follow up PR?